dashboard/app/controllers/levels_controller.rb

```diff
@@ -242,6 +242,7 @@ def level_params
       :published,
       {poems: []},
       {concept_ids: []},
+      {level_concept_difficulty_attributes: [:id] + LevelConceptDifficulty::CONCEPTS},
```

Comment:
There's not a ton of reason either way. I like LevelConceptDifficulty because we are specifically interacting with level concept difficulties here. It's semantically closer.